dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Account/Index.cshtml.cs

```diff
@@ -71,6 +73,8 @@ public async Task OnGet()
 
             OfficialName = $"{dqtUser.FirstName} {dqtUser.LastName}";
             DqtDateOfBirth = dqtUser.DateOfBirth;
+            PendingDqtNameChange = dqtUser.PendingNameChange;
+            PendingDqtDateOfBirthChange = dqtUser.PendingDateOfBirthChange;
```

Comment:
Was written as PendingDqtDateOfBirth in ticket. If that's been decided can change, otherwise this seemed more consistent